src/libraries/System.Formats.Tar/tests/TarWriter/TarWriter.Tests.cs

```diff
@@ -79,6 +79,28 @@ public void Constructor_NoEntryInsertion_WritesNothing()
             Assert.Equal(0, archiveStream.Length);
         }
 
+        [Fact]
+        public void Write_To_UnseekableStream()
+        {
+            MemoryStream stream = new MemoryStream();
```

Comment:
Nit, when you wrap a stream, might be clearer to name the inner one something like 'inner' not 'stream'.

Reply:
Fixed.